csharp/Clean.cmd

```diff
@@ -1,3 +1,3 @@
-FOR /F "tokens=*" %%G IN ('DIR /B /AD /S bin') DO RMDIR /S /Q "%%G"
```

Comment:
the existing clean.cmd just delete sub-folders and files under "bin" and "obj", but it do not delete 'bin' and 'obj' folder self. So after running clean.cmd, you still can see these two folders that we want to clean up.